docs/buildah-from.md

```diff
@@ -0,0 +1,40 @@
+## buildah-from "March 2017"
+
+## NAME
+buildah from - Creates a new working container, either from scratch or based on the provided image name as a starting point.
+
+
+## SYNOPSIS
+**buildah** **from** --image *imagename* [*options* [...]] 
+
+## DESCRIPTION
+Creates a working container based upon the supplied image name.  If the supplied image name is "scratch" a new default container will be created.
```

Comment:
Creates a  container image based on the supplied image name.   If the supplied image name is "scratch" a new default empty container image will be created.